### ISSUE TITLE
automatically create agenda for next week's learning meeting

### ISSUE THREAD

**Issue by locks** [emberjs/core-notes#353]
The learning team would like to automate the creation of the following week's agenda as a quality of life improvement.
This can be done as a GitHub action that is triggered once the current agenda is merged.
The action would create a new PR with the agenda in the correct place and the relevant date.

**Comment by MinThaMie** [comment]
I think I've found a GitHub action that would work: https://jasonet.co/posts/scheduled-actions/#jasonetcocreate-an-issue
